dpypelines/pipeline/generic_file_ingress_v1.py

```diff
@@ -0,0 +1,184 @@
+import os
+
+from dpytools.http.upload import UploadServiceClient
+from dpytools.logging.logger import DpLogger
+from dpytools.stores.directory.local import LocalDirectoryStore
+
+from dpypelines.pipeline.shared.email_template_message import file_not_found_email
+from dpypelines.pipeline.shared.notification import (
+    BasePipelineNotifier,
+    notifier_from_env_var_webhook,
+)
+from dpypelines.pipeline.shared.pipelineconfig.matching import (
+    get_required_files_patterns,
+)
+from dpypelines.pipeline.shared.utils import (
+    get_email_client,
+    get_submitter_email,
+    str_to_bool,
```

Comment:
str_to_bool was recently moved to a different place. You'll need to merge in main. You can now import in from dpytools like this: `from dpytools.utilities.utilities import str_to_bool`

Reply:
Done